cookbook/security/custom_provider.rst

```diff
@@ -9,7 +9,10 @@ When a user submits a username and password, the authentication layer asks
 the configured user provider to return a user object for a given username.
 Symfony then checks whether the password of this user is correct and generates
 a security token so the user stays authenticated during the current session.
-Out of the box, Symfony has an "in_memory" and an "entity" user provider.
+Out of the box, Symfony has three user providers: ``in_memory``, ``entity`` and 
+``chain``. The chain user provider can be used to chain several user providers 
+in case a user can be retrieved from different locations.
```

Comment:
I am not sure if we really need to explain the chain user provider here. Don't we already explain all three in the Security chapter of the book? If we wanted add the description here, I would vote for providing a short description for all three providers.